includes/sql-schema/update.php

```diff
@@ -153,6 +153,9 @@
         $db_rev = $filename;
         if ($insert) {
             dbInsert(array('version' => $db_rev), 'dbSchema');
+            if ($db_rev >= 6) {
+                $insert = 0;
```

Comment:
Eh, we're in a loop at this stage for each file, so we set $insert = 0 when we hit $db_rev 6 and then on the next file we don't insert anymore - we update.